frontend.java

```diff
@@ -0,0 +1,540 @@
+					}else{
+						if(upperLimit(accountAmount)==false){
+							frontend.error("Error: Invalid account amount. The amount is too high for the type of login.");
+						}else{
+							//convert amountStr to int well more validate
+							//make sure the account number is long enough
+							//Does there have to be an account name?
+							//make sure its a valid account
+							
+							accountAmount = addZeros(accountAmount, 8);//adds zeros in front if size is less than 8
+							accountNumber = addZeros(accountNumber, 6);//adds zeros in front if size is less than 6
+							
+							String str = "01_" + accountNumber + "_" + accountAmount; 
+							frontend.transactionFile.add(str);
+							//write.writeToSummaryFile(str);
+							System.out.println("Successful Deposit.");
+						}
+					}
+				}
+			}
+		}
+	}
+	public static void front_transfer() {
+		//get account number
+		//get amount
+		System.out.print("To Account Number:");
+		String toAccountNumber = frontend.input();
+		if(accountNumber(toAccountNumber)==false){
+			frontend.error("Error: Invalid 'to' account.");
+		}else{
+			System.out.print("From Account Number:");
+			String fromAccountNumber = frontend.input();
+			if(accountNumber(fromAccountNumber)==false){
+				frontend.error("Error: Invalid 'From' account.");
+			}else{
+				System.out.print("Account Amount:");
+				String accountAmount = frontend.input();
+				if(accountAmount(accountAmount)==false){
+					frontend.error("Error: Invalid amount.");
+				}else{
+
+					if(lowerLimit(accountAmount)==false){
+						frontend.error("Error: Invalid account amount. Must be greater than 0.");
+					}else{
+						if(upperLimit(accountAmount)==false){
+							frontend.error("Error: Invalid account amount. The amount is too high for the type of login.");
+						}else{
+							//convert amountStr to int well more validate
+							//make sure the account number is long enough
+							//Does there have to be an account name?
+							//make sure they are both valid accounts
+								
+							accountAmount = addZeros(accountAmount, 8);//adds zeros in front if size is less than 8
+							fromAccountNumber = addZeros(fromAccountNumber, 6);//adds zeros in front if size is less than 6
+							toAccountNumber = addZeros(toAccountNumber, 6);//adds zeros in front if size is less than 6
+							
+							
+							String str = "03_" + toAccountNumber + "_" + fromAccountNumber + "_" + accountAmount; 
+							frontend.transactionFile.add(str);
+							//write.writeToSummaryFile(str);
+							System.out.println("Successful Transfer.");
+						}
+					}
+				}
+			}
+		}
+	}
+	public static boolean accountName(String name){
+		//name
+		//Make sure the whole name is letters.
+		boolean valid = true;
+		for (char ch : name.toCharArray()){
+	        int asciiVal = (int)ch;	
+			//if((65 <= asciiVal <= 90) || (97 <= asciiVal <= 122) || (asciiVal == 32)){
+			if((asciiVal >= 65) || (asciiVal <= 90) || (asciiVal >= 97) || (asciiVal <=122)){
```

Comment:
While I can understand what you're doing, partly via code, partly via comments, this loop is clunky.

Replace it with Regex - http://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html

So you'd be checking whether the string matches using regex, which is more powerful and flexible.